### ISSUE TITLE
EB unknown-issuer page misses detailed information

### ISSUE THREAD

**Issue by thijskh** [OpenConext/OpenConext-engineblock#131]
This page used to show details like IP, time, local and remote entityID's, and tracking ID but does not anymore for me.
```
https://engine.surfconext.nl/authentication/feedback/unknown-issuer?entity-id=https%3A%2F%2Fexample.org&destination=https%3A%2F%2Fengine.surfconext.nl%2Fauthentication%2Fidp%2Fsingle-sign-on%2Fkey%3Adefault
```

**Comment by thijskh** [comment]
Same for VO membership required.

![surfdrive](https://cloud.githubusercontent.com/assets/3808792/7294663/ab2381da-e9af-11e4-9a3f-d8691e8f0593.png)

**Comment by thijskh** [comment]
Fixed in 4.6.
